### PR TITLE
Upgrade kinto signer 1.3.2

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
The build hang I don't know why: https://travis-ci.org/mozilla-services/kinto-dist/builds/213006673

File: smoke-test.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ echo '{"data": {"type": "logo"}}' | http --check-status PUT $SERVER/buckets/blog
 http --check-status --form POST $SERVER/buckets/blog/collections/articles/records/logo/attachment attachment@kinto-logo.svg --auth $AUTH
 
 # kinto-signer test
-curl -O https://raw.githubusercontent.com/Kinto/kinto-signer/1.3.0/scripts/e2e.py
```
Comment (leplatrem) [inline smoke-test.sh]: note: there was no change between those 2 files

**Comment by leplatrem** [comment]
The problem comes from kinto-signer `listeners.send_signer_events()` which hangs when sending a `ResourceChanged` event.

**Comment by leplatrem** [comment]
Since some database operations are performed in `AfterResourceChanged` we should commit the changes (`transaction.get_current().commit()` or plug in beofore commit hook instead of `AfterResourceChanged`)

**Pull request merged**